server/server.go

```diff
@@ -74,14 +74,15 @@ const (
 
 // HTTPServer is an implementation of Server interface
 type HTTPServer struct {
-	Config            Configuration
-	InfoParams        map[string]string
-	ServicesConfig    services.Configuration
-	amsClient         *amsclient.AMSClient
-	GroupsChannel     chan []groups.Group
-	ErrorFoundChannel chan bool
-	ErrorChannel      chan error
-	Serv              *http.Server
+	Config                 Configuration
+	InfoParams             map[string]string
+	ServicesConfig         services.Configuration
+	amsClient              amsclient.AMSClient
+	GroupsChannel          chan []groups.Group
+	ErrorFoundChannel      chan bool
+	ErrorChannel           chan error
+	useOrgClustersFallback bool
```

Comment:
ah should not it be part of `Config`?

Reply:
Mmm, and in fact, it is! It is really duplicated. I will push a new commit. Good catch!